tiller_python_service/error_handlers.py

```diff
@@ -5,6 +5,29 @@
 
 LOGGER = logging.getLogger(__name__)
 
+def my_new_sorting_function(arr):
+    n = len(arr)
```

Comment:
This new sorting function is implemented very inefficiently, can we replace it with a merge sort? help tiller